pkg/azureclients/diskclient/azure_diskclient.go

```diff
@@ -64,11 +64,7 @@ func New(config *azclients.ClientConfig) *Client {
 		apiVersion = AzureStackCloudAPIVersion
 	}
 
-	if config.ExtendedLocation != nil {
-		apiVersion = AzureEdgeZonesAPIVersion
-	}
-
-	klog.V(2).Infof("Azure DisksClient using API version: %s", AzureEdgeZonesAPIVersion)
+	klog.V(2).Infof("Azure DisksClient using API version: %s", apiVersion)
```

Comment:
`2020-12-01` is available on all regions including gov cloud?

Reply:
should be very cautious about this version change unless we could make sure it's available on all regions otherwise there could be break

Reply:
yes, available now in all clouds.